cmd/fission-cli/app/app.go

```diff
@@ -86,7 +86,8 @@ func App() *cobra.Command {
 	})
 
 	wrapper.SetFlags(rootCmd, flag.FlagSet{
-		Global: []flag.Flag{flag.GlobalServer, flag.GlobalVerbosity, flag.KubeContext},
+		Global:   []flag.Flag{flag.GlobalServer, flag.GlobalVerbosity, flag.KubeContext},
+		Optional: []flag.Flag{flag.PreCheckOnly},
```

Comment:
I don't think the flag is required here because it would just show up in the fission help which wouldn't make sense. This flag is already included in `fission check -h` so you can remove this from here.